source/particle/property/interface.cc

```diff
@@ -67,7 +65,7 @@ namespace aspect
       InitializationModeForLateParticles
       Interface<dim>::late_initialization_mode () const
       {
-        return initialize_to_zero;
+        return interpolate;
```

Comment:
Yes this is intended, because it is the far more reasonable default. It is only a problem if somebody implemented their own particle property plugin so far, and did not implement this function (which I seriously doubt). All included plugins are updated accordingly.